tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/HttpMessageHandlerTests.cs

```diff
@@ -41,7 +42,8 @@ public HttpMessageHandlerTests(ITestOutputHelper output)
             from instrumentationOptions in InstrumentationOptionsValues
             from socketHandlerEnabled in new[] { true, false }
             from queryStringEnabled in new[] { true, false }
-            select new object[] { instrumentationOptions, socketHandlerEnabled, queryStringEnabled };
+            from traceId128Enabled in new[] { true, false }
```

Comment:
I'm touching this file as well. I can wait to merge if you think we can merge in the coming days.

Reply:
Thanks for the heads up. I fixed the merge conflict after you merged your changes.

Reply:
Oh, `metadataSchemaVersion` was @zacharycmontoya, not you :P